virtool/jobs/api.py

```diff
@@ -190,13 +190,23 @@ async def put(self, job_id: str, /) -> Union[r200[JobResponse], r403, r404, r409
         try:
             document = await get_data_from_req(self.request).jobs.cancel(job_id)
         except ResourceNotFoundError:
-            raise NotFound
+            raise NotFound()
         except ResourceConflictError:
             raise HTTPConflict(text="Job cannot be cancelled in its current state")
 
         return json_response(document)
```

Comment:
This should return only the ping object to stick to RESTful semantics.

Requests to `/jobs/:id` will return a job. Requests to `/jobs/:id/ping` will return a ping.